ddterm/com.github.amezin.ddterm.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=Drop Down Terminal
+NoDisplay=true
+Comment=Another drop down terminal extension for GNOME Shell. With tabs. Works on Wayland natively.
+Icon=terminal
+Exec=gdbus call --session --dest org.gnome.Shell --object-path /org/gnome/Shell/Extensions/ddterm --method com.github.amezin.ddterm.Extension.Toggle
+Actions=
+Categories=Utility;
+DBusActivatable=true
```

Comment:
If I'm not mistaken, to support D-Bus activation the application must own a well-known name and it must have a service description file: https://dbus.freedesktop.org/doc/dbus-specification.html#message-bus-starting-services

And `org.freedesktop.Application` isn't implemented by the extension either https://specifications.freedesktop.org/desktop-entry-spec/1.1/ar01s07.html

ddterm inherits `org.freedesktop.Application` implementation from GLib/GApplication... But ddterm app shouldn't be started directly.

Although, maybe it's possible to also install a D-Bus service description file, in the same way as the .desktop file (does D-Bus have per-user service directories?). And write the service description in a way that launches ddterm app correctly (i. e. through the extension).

But I'd recommend to experiment with D-Bus activation in a separate pull request anyway.

Reply:
Yep, that's definitely a story for another day 😄
But I like the idea, can investigate further later!

Reply:
If D-Bus activation isn't supported, I think `DBusActivatable=true` should be removed

Reply:
Opsie, I thought I had deleted the commit that made that change. Fixed now.